Changelog.md

```diff
@@ -15,6 +15,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   
 ### Fixed
 
+- Fixed bug that caused operator implementations to appear in the documentation
+  although they should be omitted because of their lower access level.
+  #264 by @Lukas-Stuehrk
```

Comment:
```suggestion
- Fixed links to type declarations.
  #264 by @Lukas-Stuehrk.
- Fixed bug that caused operator implementations to appear in the documentation
  although they should be omitted because of their lower access level.
  #264 by @Lukas-Stuehrk
```